# Exception handeling

```
try:
    something()
except:
    print("error")
```
This is naughty on so many levels. 

- Catch all `except` blocks are almost always a terribl idea 
- python's built-in traceback is super informative. Don't throw away the information. We like information. 

To learn more about exception handling read this wonderful article: https://www.codementor.io/sheena/python-exception-handling-ogr0a41t7 (shameless plug)

